fakeredis/_server.py

```diff
@@ -23,10 +22,16 @@
 
 LOGGER = logging.getLogger('fakeredis')
 REDIS_LOG_LEVELS = {
-    b'LOG_DEBUG': logging.DEBUG,
-    b'LOG_VERBOSE': logging.INFO,
-    b'LOG_NOTICE': logging.INFO,
-    b'LOG_WARNING': logging.WARNING
+    b'LOG_DEBUG': 0,
+    b'LOG_VERBOSE': 1,
+    b'LOG_NOTICE': 2,
+    b'LOG_WARNING': 3
+}
+REDIS_LOG_LEVELS_TO_LOGGING = {
+    0: logging.DEBUG,
+    1: logging.INFO,
+    2: logging.INFO,
+    3: logging.WARNING
 }
```

Comment:
Is the double indirection because redis.LOG_DEBUG has the value 0 in real redis?

Reply:
Yes, and because user can actually use numbers instead of `redis.LOG_*` variables, just like in `logging`, but level values are different